commcare_connect/opportunity/views.py

```diff
@@ -66,7 +66,10 @@ def get_success_url(self):
 
     def form_valid(self, form):
         form.instance.modified_by = self.request.user.email
-        return super().form_valid(form)
+        response = super().form_valid(form)
+        add_connect_users.delay(form.cleaned_data["users"], form.instance)
```

Comment:
I don't think you can pass a file to a celery task like this, also if you want the actual file you'll need to do `request.FILES["users"]`.

If you want to give celery access to the file you'll need to save it using Django storage backend:

```
name = uuid4().hex
storages["default"].save(name, request.FILES["users"])
```